src/blocks/ImageTextBlockBrandPivot/index.tsx

```diff
@@ -317,6 +318,7 @@ export const ImageTextBlockBrandPivot: React.FunctionComponent<ImageTextBlockPro
         index={index}
         brandPivot
         fullWidth={full_width}
+        padding={background_type === 'video' ? 'relaxed' : 'minimal'}
```

Comment:
I think you know more about this @gustaveen but shouldn't the `'relaxed'` padding also be used if we use an image and not a video as background? Or am I missing something?

Reply:
My idea was the change as little as possible from the current implementation. That's why I try to target only when the block is used with a video element.